src/models/job.jsx

```diff
@@ -33,6 +36,30 @@ const JobContext = createContext({
   countJobs: (_queryConstraints) => Number,
 });
 
+const updateJob = async (jobId, payload) => {
+  // remove the jobId field if existed. No need to put the doc id into the doc data
+  if (payload.id) {
+    payload.id == undefined;
```

Comment:
Sorry I didn't see this when I initially looked through, but should this be `payload.id = undefined;` @makersmelx

Reply:
you are right. I need to fix it.